testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -5809,6 +5809,7 @@ public void testDataMappingSmokeTest()
 
     private void testDataMapping(DataMappingTestSetup dataMappingTestSetup)
     {
+        log.info("dataMappingTestSetup=%s".formatted(dataMappingTestSetup));
```

Comment:
@krvikash I've updated PR after you approved.
As in trino parametrized tests are not used any more it's not clear for what test case the test fails if it fails in any place of the test.

Reply:
how about
```suggestion
        log.info("dataMappingTestSetup=%s", dataMappingTestSetup)
```